### ISSUE TITLE
Financetime zoomable domain returns values off by 0.52 on both ends

### ISSUE THREAD

**Issue by DBauer15** [andredumas/techan.js#182]
Hello,
I am using techan in a plot that displays finance data, therefore I use the financetime scale. The project offers users to select a brushed area by setting a brush in a separate context plot. If the range of values changes in the y axis (which displays prices) I recalculate the y domain using the currently active datapoints selected by my brush. Here's where I have several issues.

Having a data set of 550 values and setting no zoom at all, calling the function 
`xScale.zoomable().domain()` 
(xScale is of type techan.scale.financetime) returns 
`[-0.52, 549.5199999999999]` 
which is all fine except that the array should range from 0 to 550. 

Why are these values bounded by 0.52 on each end respectively? It also happens if I narrow down my selection to just a few data points.

Having to work with these domains while setting up zoomed areas in my plot, I find myself correcting these issue over and over again. 

It's especially bothersome to calculate a new y domain (let's assume the y axis is based on a linear scale). I do it like this:

`this.yScale.domain(techan.scale.plot.ohlc(plotdata.slice(this.xScale.zoomable().domain()[0]+1,
this.xScale.zoomable().domain()[1]+1), this.plotFocus.accessor()).domain());`

As you can see I have to add 1 to every end of the domain to correctly include all the values currently displayed.
Is there any other way to do this conveniently?

## TL;DR:
Are the values of financetime.zoomable.domain deliberately off by 0.52 on each end of the range or is it a bug and what ways are there to avoid struggling with this offset everytime you want to calculate anything relating to zoom and brushing?
